Windows/MainWindow.cpp

```diff
@@ -880,9 +880,10 @@ namespace MainWindow
 			break;
 
 		case WM_MENUSELECT:
-			// Unfortunately, accelerate keys (hotkeys) shares the same enabled/disabled states
```

Comment:
Do hotkeys not matter?  What if I press a hot key but don't open the menu first?  This comment seems to suggest it's an issue.

-[Unknown]